test/images/resource-consumer/BASEIMAGE

```diff
@@ -1,4 +1,4 @@
-amd64=gcr.io/google-containers/ubuntu-slim-amd64:0.12
```

Comment:
as noted in the other PR, maybe update the `Dockerfile` to use `clean-install` instead of `apt-get` (e.g. https://github.com/kubernetes/kubernetes/blob/33d555ef00ab3366744690631f9cf526651f7d7c/build/debian-iptables/Dockerfile#L21-L24)